packages/presentation/pubspec.yaml

```diff
@@ -14,6 +14,11 @@ dependencies:
     sdk: flutter
   domain:
     path: ../domain
+  data:
+    path: ../data
+  
+  get_it: ^7.2.0
+  injectable: ^1.5.3
```

Comment:
no need in specifying dependencies in each module. u can specify it only in one and other modules, which include the module with dependency will also have this dependency. so, if u'll add get_it to the domain, u'll have access to it in data, domain and presentation. read about transitive dependencies in the flutter